nukeThread.js

```diff
@@ -171,7 +171,7 @@ document.addEventListener('DOMContentLoaded', function () {
     document.getElementById('thread-url-box').value = url;
     document.getElementById('preserve-distinguished-checkbox').checked = parsedState.preserveDistinguished;
     document.getElementById('preserve-removed-checkbox').checked = parsedState.preserveRemoved;
+    document.getElementById("to-nuke").value = parsedState.toNuke;
     var toNuke = document.getElementById("to-nuke").value;
     nukeThread(url, toNuke);
-  }
-});
\ No newline at end of file
+  }
```

Comment:
Was removing the last line intentional? It seems like this will be a syntax error.